administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -54,16 +54,16 @@
 	// Attach behaviour to toggle button.
 	$('body').on('click', '#toogle-left-panel', function()
 	{
-		referenceHide = $('#adminForm').attr('data-hide-reference');
-		referenceShow = $('#adminForm').attr('data-show-reference');
+		referenceHide = $(this).attr('data-hide-reference');
+		referenceShow = $(this).attr('data-show-reference');
```

Comment:
vannila js
this.getAttribute('data-hide-reference');
this.getAttribute('data-show-reference');